hack/generate-aws-permissions.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
needs to use `go generate`

Reply:
> needs to use `go generate`

Is calling out to the shell script acceptable, just with a `go generate` trigger?  Or do I need to port the regexp extractor from shell to Go too?

Reply:
former

Reply:
Added a `go:generate` comment with 78f39f075 -> eb892c7d7.